BL7011/file_processing.py

```diff
@@ -142,7 +151,7 @@ def get_single_ccd_image(
         norm_factor = h5_labview_db['XS111LeftBladecurrent_diode'][index]
     elif 'i0 rlrl' in correction:  # XS111 RLRL diode normalization
         print(h5_labview_db['XS111RLRL_diode'].shape)
-        norm_factor = -h5_labview_db['XS111RLRL_diode'][index]
+        norm_factor = np.abs(h5_labview_db['XS111RLRL_diode'][index])
         print(norm_factor)
```

Comment:
using `if verbose:` for print statements could be cleaner for the methods

Reply:
Implemented a verbose statement for value printing in [e7329a9]